_posts/2016/2016-03-16-team-orange-mallorca.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "A team of ePagees conquers Tierra Buena in Mallorca"
```

Comment:
The title is very nice but I think it just fits for internal readers. Maybe you find a SEO-friendly one that also attracts external readers?